modules/kubernetes-addons/aws-pca-issuer/variables.tf

```diff
@@ -0,0 +1,46 @@
+
+variable "helm_config" {
+  type        = any
+  default     = {}
+  description = "AWS PCA Issuer Helm Config"
+}
+
+variable "manage_via_gitops" {
+  type        = bool
+  default     = false
+  description = "Determines if the add-on should be managed via GitOps."
+}
+
+variable "irsa_iam_permissions_boundary" {
+  type        = string
+  default     = ""
+  description = "IAM Policy ARN for IRSA IAM role permissions boundary"
+}
```

Comment:
Needs to be updated if [PR](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/pull/351) is merged before this one.